public/index.html

```diff
@@ -22,6 +22,14 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>React App</title>
+    <style>
+      * {
+        padding: 0;
+        margin: 0;
+        box-sizing: border-box;
+        font-family: 'Roboto', 'Helvetica', 'Arial', sans-serif;
```

Comment:
You can also handle globals with styled-components: https://styled-components.com/docs/api#createglobalstyle